### DISCUSSION TITLE
quiet sound and dual battarey

### DISCUSSION THREAD

**Discussion by Pavlutioni** [MultimediaLucario/Lenovo-ThinkPad-T440s#35] [Q&A]
Hello. I have t440s. I did everything according to the instructions and everything works for me. But I only have one battery displayed and very quiet sound. I have Big Sur installed. Help me solve these two problems, please.

**Comment by 4mitabh** [comment]
MacOS supports only one battery... so, while the other battery will work, there is no way to see it's individual status. What you see will be combined status of both batteries.

The low volume is strange... maybe try updating to the latest drivers ?
